build.xml

```diff
@@ -12,8 +12,8 @@
 	</path>
 	<target name="init">
 		<mkdir dir="${builddir}/bin/resources"/>
-		<copy includeemptydirs="false" todir="${builddir}/bin">
-			<fileset dir="src" excludes="**/*.launch, **/*.java"/>
+		<copy includeemptydirs="false" todir="${builddir}/bin/org">
```

Comment:
I don't know a single thing about ant builds, but why is the destination directory changed to `bin/org` here?

Reply:
Previously we could copy `src/*` into `bin/` to get `bin/org`.  Since `org` is moving to the top level, we need to copy `org/*` into `bin/org/`.